### DISCUSSION TITLE
LAN-C IP address - “invalid host”  when trying to add  (yet router shows this is the correct IP address) - what to do now

### DISCUSSION THREAD

**Discussion by LancevdMeer** [michaelarnauts/home-assistant-comfoconnect#30] [Q&A]
Hi Michael + Community

Thank you for the great work done. I followed all instructions by the letter (HA VM Linux version, on a MAC with HACS), and used HACS to get the repository added. However - the last step is failing :

when I want to add the integration,I get “invalid host” when entering the LAN-C IP address

As you can see from my router screenshot, I am using  IP which MAC address matches that printed on the LAN-C

I have factory reset my LAN-C to make sure the gateway is on default pwd 0000, and am able though the iOS comfocontrol app to access my ComfoAirQ450 from external internet (eg from 4G eg outside my network). Hence, the router and internet connect work to the LAN-C

My home-assist install has no issues to connect with other devices on my lan network, nor on my wifi. The Mac being used is lan-connected to my router.

Nevertheless, I keep getting “invalid host”

Please advice

![IMG_7579](https://github.com/michaelarnauts/home-assistant-comfoconnect/assets/42781470/e6ac23c3-b0d6-4303-aad8-a39e96e2c514)
![IMG_7578](https://github.com/michaelarnauts/home-assistant-comfoconnect/assets/42781470/65bdda8d-9c40-4b26-aeab-66c13c83f9ae)

**Comment by LancevdMeer** [comment]
And there we have it: the Zehnder LAN C Ethernet cable cannot be on a switch but must be directly plugged into the main router for homeassist to notice and connect

**Reply by michaelarnauts** [reply]
Then your switch isn't doing L2 switching, but is probably a router. Discovery of the Comfoconnect is using broadcasts so if needs to be in the same network.

I don't understand why the app is able to find it though.

**Reply by LancevdMeer** [reply]
Hi Michael, thank you for the reply. 

The switch is an unmanaged SBVR - SP1080G - Ethernet Switch. I was plugged into the main router, and the Comfoconnect and Mac Server were both connected to the switch, yet comfoconnect remailed undiscovered. 

When I plugged my Mac server (with HA on Linux VM) in the main router as wel the Zehnder LAN C cable, the HA app picket the Zehnder’s presence up. 

I ended up btw with assigning the wifi as network to the VM. And used the LAN C to connect with the HA WiFi connect. This works smoothly and improved the connection with Alexa and Google Speaker (Google Home).

**Comment by LancevdMeer** [comment]
Yet @michaelarnauts , I do have one weird additional integration issue where Alexa is able to change the fan speed yet Google Home is not. 

My main goal of the comfoconnect integration is to use voice commands from my Alexa and Google devices to increase the fan speed when I am cooking (in order to drain the excess humidity). 

The pictures attached show this clearly. When creating a routing on Alexa, the first two picture allow a preset percentage, 

Google Home only allows the on/off option as selectable and NOT a percentage (last three pictures) when  creating an automation. 

(Note both have a clima group option that does allow the speed options, it just seem to miss in the visible options in Google home when setting up a google automation. 

Any thoughts here ?

Thank you !

![IMG_7652](https://github.com/michaelarnauts/home-assistant-comfoconnect/assets/42781470/01037fcf-ba33-44b2-8f67-93f232090f43)
![IMG_7653](https://github.com/michaelarnauts/home-assistant-comfoconnect/assets/42781470/fc7628f7-2b97-4611-a95a-40ba686e4a36)
![IMG_7654](https://github.com/michaelarnauts/home-assistant-comfoconnect/assets/42781470/77543247-242e-446a-b877-dc302905dc44)
![IMG_7655](https://github.com/michaelarnauts/home-assistant-comfoconnect/assets/42781470/65798182-a988-4f61-921d-47a643c0946b)
![IMG_7658](https://github.com/michaelarnauts/home-assistant-comfoconnect/assets/42781470/5e2c580c-0279-4889-b092-a7cde249902c)

**Comment by yann1420** [comment]
![Zehnder ComfoAir Q600](https://github.com/user-attachments/assets/6b64badf-9377-48ec-b9e1-613580acee3e)

For me to understand, is the RJ45 connector (photo) part of the **ComfoConnect LAN bridge**?
Thanks in advance, Yann
